### ISSUE TITLE
Wrong field to save IGNORE_CASE env var

### ISSUE THREAD

**Issue by bkatrenko** [tihardc/Gomework-Grep#5]
https://github.com/tihardc/Gomework-Grep/blob/53b64265dd544afb783a6e571ce36637ad78d78b/conf.go#L37

The idea is that it is confusing for devs when we save the variable "IGNORE_CASE" to field option inside the structure. I would say we must rename IGNORE_CASE to option than

**Comment by tihardc** [comment]
Yes, absolutely, and if I'd thought it was worth your time pretending to be
a user I'd have talked to you about that before sending it in. I think I
did mention it in the readme. And you can see it does do regexes!
